tests/test_components.py

```diff
@@ -84,3 +86,23 @@ def test_real_estate_area():
     RealEstateArea(
         value=123.45, unitCode="SQMT", raw="123.45 square meters", areaType="LOT"
     )
+
+
+def test_headers():
+    Header(name="Connection", value="keep-alive")
+    Header(name="Content-Type", value="text/html; charset=utf-8")
+
+
+def test_navigation():
+    Navigation(url="https://example.com")
+    headers = [
+        Header(name="Content-Type", value="application/x-www-form-urlencoded"),
+        Header(name="Host", value="foo.example"),
+    ]
+    Navigation(
+        name="Post Test",
+        url="https://example.com/test",
```

Comment:
Let's also add a test that would try out `_Url` conversion with `url_to_str`

Reply:
Done